numpy/lib/ufunclike.py

```diff
@@ -37,6 +38,11 @@ def func(x, out=None, **kwargs):
     return func
 
 
+def _fix_dispatcher(x, out=None, **kwargs):
+    return (x, out)
+
+
+@array_function_dispatch(_fix_dispatcher, verify=False)
 @_deprecate_out_named_y
```

Comment:
Should this be added to the dispatcher?

Reply:
Yes, this is a better idea. That way the warning will be raised even if the function is dispatching.